lib/source-destination/configured-source/configured-source.ts

```diff
@@ -79,10 +79,10 @@ export class SourceDisk extends Disk {
 	}
 
 	protected async _write(
-		_buffer: Buffer,
-		_bufferOffset: number,
-		_length: number,
-		_fileOffset: number,
+		_buffer: Buffer, // eslint-disable-line @typescript-eslint/no-unused-vars
+		_bufferOffset: number, // eslint-disable-line @typescript-eslint/no-unused-vars
+		_length: number, // eslint-disable-line @typescript-eslint/no-unused-vars
+		_fileOffset: number, // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
It's fine to drop those, since we & TS will be checking the base `Disk` interface for the exact types of this.

Reply:
it's not a type problem, the problem is that they're not used and not optional